include/nbl/video/IGPUImage.h

```diff
@@ -28,8 +27,9 @@ class IGPUImage : public asset::IImage, public IDeviceMemoryBacked, public IBack
 		};
 		struct SCreationParams : asset::IImage::SCreationParams, IDeviceMemoryBacked::SCreationParams
 		{
-			E_TILING tiling = ET_OPTIMAL;
-			E_LAYOUT initialLayout = EL_UNDEFINED;
+			E_TILING tiling : 1 = ET_OPTIMAL;
+			// No `initialLayout` due to https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkImageCreateInfo.html#VUID-VkImageCreateInfo-initialLayout-00993
+			uint8_t preinitialized : 1 = false;
```

Comment:
finally, much nicer.
this is one of the changes that will cause compile errors, but worth it